apps/src/templates/instructions/ImmersiveReaderButton.jsx

```diff
@@ -3,12 +3,17 @@ import PropTypes from 'prop-types';
 import handleLaunchImmersiveReader from '@cdo/apps/util/immersive_reader';
 import {renderButtons} from '@microsoft/immersive-reader-sdk';
 import cookies from 'js-cookie';
-import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import classNames from 'classnames';
 
 class ImmersiveReaderButton extends Component {
   static propTypes = {
     title: PropTypes.string,
-    text: PropTypes.string
+    text: PropTypes.string,
+    isImmersiveButtonHasRoundBorders: PropTypes.bool,
```

Comment:
nit: since this is a property on `ImmersiveReaderButton` can we simplify this prop name to `hasRoundBorders`?

Reply:
Nice catch, will fix!